pycaret/time_series/forecasting/oop.py

```diff
@@ -233,22 +234,20 @@ def _get_metrics(self, raise_errors: bool = True) -> dict:
     def _get_default_plots_to_log(self) -> List[str]:
         return ["forecast", "residuals", "diagnostics"]
 
-    def _check_fh(
-        self, fh: Union[List[int], int, np.array, ForecastingHorizon]
-    ) -> Union[np.array, ForecastingHorizon]:
+    def _check_fh(self, fh: PyCaretForecastingHorizonTypes) -> ForecastingHorizon:
         """
         Checks fh for validity and converts fh into an appropriate forecasting
         horizon compatible with sktime (if necessary)
 
         Parameters
         ----------
-        fh : Union[List[int], int, np.array, ForecastingHorizon]
-            Forecasting Horizon
+        fh : PyCaretForecastingHorizonTypes
+            `PyCaret` compatible Forecasting Horizon
```

Comment:
Double backtics should be used in documentation to signify code instead of single backtics (RST sphinx syntax)